model/entities/balance.py

```diff
@@ -5,6 +5,8 @@
 Balance(celo=2, cusd=10) + Balance(celo=5, cusd=0) = Balance(celo=7, cusd=10)
 """
 from typing import Callable, Dict, TYPE_CHECKING
+
+from model.types import Fiat, Pair
```

Comment:
this is causing a circular import error

Reply:
Should be resolved with the latest commit